Processing/AppDelegate.swift

```diff
@@ -16,6 +17,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        FirebaseApp.configure()
+        UINavigationBar.appearance().tintColor = UIColor.white
```

Comment:
Maybe put `UIColor.white` also in the `UIColor extension` as `processingTintColor`?